src/main/proto/netflix/titus/titus_autoscale_api.proto

```diff
@@ -54,6 +54,8 @@ message AlarmConfiguration {
   // The statistic for the metric associated with the alarm, other than
   // percentile.
   oneof statisticOneof { Statistic statistic = 8; }
+
+  repeated MetricDimension metricDimensions = 9;
```

Comment:
Can you add a comment to this field? All of the other metrics have a comment and I don't think it would be clear to a user if/what they should set here. I believe the other comments were derived from the comments on their AWS equivalents.